app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,22 @@
  */
 .cancel {
   text-align: center;
+}
+
+body {
+ background-image: image_url("");
+ background-size: cover;
+ background-repeat: no-repeat;
+ background-position: center top;
+ background-position: 0px 45px;
+}
 
+.welcome {
+  background:rgba(255,255,255,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space